meta-lmp-base/recipes-support/docker-credential-helper-fio/files/docker-credential-fio-helper

```diff
@@ -6,5 +6,9 @@ export LOG_STDERR=1
 LOGLEVEL="${CREDS_LOGLEVEL-4}"
 
 if [ "$1" = "get" ] ; then
-	exec /usr/bin/aktualizr-get --loglevel $LOGLEVEL -u https://ota-lite.foundries.io:8443/hub-creds/
+	server=$(cat /var/sota/sota.toml | grep ^server | head -n1 | cut -d\" -f2)
```

Comment:
One problem is that this helper will be called with all users, and only root will have access to sota.toml, so that will let to a permission denied issue when called as fio (or any normal user).

So I would probably move the default server settings above this line (server="https://ota-lite.foundries.io:8443") and check if enough permission to parse sota.toml, and then update the variable if another server is available.

You can also drop cat and head by having one single grep call instead, such as:

`grep -m1 ^server /var/sota/sota.toml | cut -d\" -f2
`

Reply:
i've been thinking the same thing. its really confusing right now. this would help usability.

Reply:
force push fix.